aws_auth.tf

```diff
@@ -2,9 +2,10 @@ data "aws_caller_identity" "current" {
 }
 
 locals {
+  arn_partition = data.aws_partition.current.partition
```

Comment:
We don't need this local, it's a simple assignment. I don't think it adds value.

Can you please replace `local.arn_partition` with `data.aws_partition.current.partition`

Reply:
Done